baybe/strategies/base.py

```diff
@@ -3,26 +3,48 @@
 from abc import ABC, abstractmethod
 from typing import Optional
 
+import cattrs
 import pandas as pd
 from attrs import define, field
 
-from baybe.recommenders.base import Recommender
+from baybe.exceptions import DeprecationError
+from baybe.recommenders.base import Recommender, RecommenderProtocol
+from baybe.recommenders.deprecation import structure_recommender_protocol
 from baybe.searchspace import SearchSpace
 from baybe.serialization import SerialMixin, converter, unstructure_base
-from baybe.strategies.deprecation import structure_strategy
 
 
 @define
-class Strategy(SerialMixin, ABC):
+class Strategy(SerialMixin, RecommenderProtocol, ABC):
```

Comment:
According to the intended usage of `Protocol`, it should not be necessary to use inheritance here. This might however be different compared to `Recommender` since this class lives in another file. This should be double-checked!

Reply:
keeping it, see comments in dev thread